packages-exp/firebase-exp/performance/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "firebase-exp/performance",
+  "main": "dist/index.cjs.js",
+  "browser": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "esm2017": "dist/index.esm2017.js",
```

Comment:
Remove. We don't need an esm2017 build in `firebase-exp` which will be just the same as the esm build, because we just reexport everything from `@firebase` packages